.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,2 @@
+👉 Please follow one of these issue templates https://github.com/warehouseai/warehouse.ai-status-models/issues/new/choose
+d
```

Comment:
what's the `d` doing?

Reply:
vim delete :) delete delete delete 

![](https://media.giphy.com/media/RlBatY2OuvZyAIqaks/giphy.gif)